packages/cli/tsdown.config.ts

```diff
@@ -4,7 +4,7 @@ const isProd = process.env.TSUP_DEV !== 'true';
 
 export default defineConfig({
   format: ['esm'],
-  entry: ['./src/index.ts'],
+  entry: ['./src/index.ts', './src/command-reference.ts'],
```

Comment:
Here, I prefer to generate only this tool on development. We already have the `isProd` variable, so we can push this entry only when it's false.